src/Rules/PropertyConditions/ElementGroups.cs

```diff
@@ -31,6 +31,8 @@ static class ElementGroups
         public static Condition ParentWPFDataItem = CreateParentWPFDataItemCondition();
         public static Condition WPFScrollBarPageButtons = CreateWPFScrollBarPageButtons();
         public static Condition NonFocusableSliderButtons = CreateNonFocusableSliderButtonsCondition();
+        public static Condition IsCheckBoxText = Text & AnyAncestor(CheckBox);
+        public static Condition IsWPFCheckBoxText = WPF & IsCheckBoxText;
         public static Condition NameRequired = CreateNameRequiredCondition();
```

Comment:
Order here is drive by the fact that `CreateNameRequiredCondition` depends on `IsWPFCheckBoxText`.